### ISSUE TITLE
The green badge on the menu icon is too bright

### ISSUE THREAD

**Issue by rappo** [EFForg/privacybadgerfirefox#460]
I find the green "0" badge on the Privacy Badger icon to be too bright/harsh, to the point where it's obnoxious. Can we tone it down a little?
![image](https://cloud.githubusercontent.com/assets/2245234/9124305/af0bc076-3c4a-11e5-874c-805a0a917e33.png)

The green used in the menu itself isn't bad, perhaps pick just one green to use everywhere and stick with it:
![image](https://cloud.githubusercontent.com/assets/2245234/9124334/d8ff3fac-3c4a-11e5-97c2-66ed633183e4.png)

I assume this also affects https://github.com/EFForg/privacybadgerchrome

**Comment by jdorenbush** [comment]
Agreed. The white number against that bright green background is too difficult to read...But is the "0" even necessary? Why not just use the green color to indicate no blocked items and only display a number when items are blocked?

**Comment by rugk** [comment]
Well... why not just display nothing at all. So only if something is blocked there a number is displayed.
Otherwise you're right the green is too bright.

**Comment by rhendric** [comment]
:+1: from me—IMO both the green and the red need to calm down.

If you have the [Stylish](https://addons.mozilla.org/en-US/firefox/addon/stylish/) Firefox extension, you can add this CSS to recolor the badge (this is fragile and really only meant as a temporary workaround, but it beats installing a custom build of Privacy Badger):
```css
@namespace url(http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul);

#toggle-button--jid1-mnnxcxisbpnsxqjetpack-pb-button .toolbarbutton-badge[style="background-color: rgb(255, 0, 0);"] {
  background-color: rgb(160, 0, 0) !important;
}

#toggle-button--jid1-mnnxcxisbpnsxqjetpack-pb-button .toolbarbutton-badge[style="background-color: rgb(0, 255, 0);"] {
  background-color: rgb(0, 160, 0) !important;
}
```

**Comment by cooperq** [comment]
we happily accept pull requests to fix such issues.

**Comment by fuglede** [comment]
The PR in #509 changes the colors. (I was being a dumb dumb and not checking the existing issues before submitting that.)

**Comment by cooperq** [comment]
Fixed  by #509
